### ISSUE TITLE
A multi language version ? A french version ? 

### ISSUE THREAD

**Issue by rhq** [jariz/Noti#68]
Noti is a really great tool, but that would be much better if we could choose a language.. ^^

**Comment by jariz** [comment]
Would you be willing to translate? (I'm guessing you speak french?)

**Comment by jariz** [comment]
Made a separate issue to keep track of all available translators (#69)

**Comment by rhq** [comment]
Yes, I would be glad to translate if you have a list of all the words/sentences in english. :)
I do speak french, maybe I could find help for a Spanish version too.

I just don't know how to develop mac application. Maybe you got like a json file ?

**Comment by jariz** [comment]
@rhq Yea no worries, I'll probably mail over a list of words that translators can just mail back to me again after, so no need for coding knowledge.

**Comment by rhq** [comment]
Ok cool, great !

**Comment by hazmattt** [comment]
Hey, new Github user, long time Noti user dropping in. If somebody could help me figure out why I cannot interact with my notifications from any app (reply, actions, etc), I'll be glad to pitch in with Japanese.
